### PR TITLE
Add container for eido

### PR DESCRIPTION
**Pull request by rafalstepien**
# Submitting a Container
 
 (If you're requesting for a new container, please check the procedure described [here](https://github.com/BioContainers/containers#241-how-to-request-a-container).  
 
 ## Check BioContainers' Dockerfile [specifications](https://github.com/BioContainers/specs)  
 ## Checklist
 
 1. Misc
 - [x] My tool doesn't exist in [BioConda](#make-sure-your-tool-isnt-already-present-in-bioconda)   
 - [x] The image can be built   
 
 2. [Metadata](#check-biocontainers-dockerfile-metadata)  
 - [x] LABEL base_image  
 - [x] LABEL version
 - [x] LABEL software.version  
 - [x] LABEL about.summary  
 - [x] LABEL about.home  
 - [x] LABEL about.license   
 - [x] MAINTAINER  
 
 3. Extra (optionals)
  - [ ] I have written tests in test-cmds.txt  
  - [ ] LABEL extra.identifier    
  - [ ] LABEL about.documentation   
  - [ ] LABEL about.license_file
  - [ ] LABEL about.tags   
 
 
 ## Check [BioContainers'](https://github.com/BioContainers/specs) Dockerfile metadata

### CODE REVIEW

**Comment by rafalstepien** [comment]
Container request: https://github.com/BioContainers/containers/issues/458

File: eido/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Base Image
+FROM biocontainers/biocontainers:latest
+
+# Metadata
+LABEL base_image="biocontainers:latest"
+LABEL version="3"
```
Comment (osallou) [inline eido/Dockerfile]: Should be version 1

**Review by osallou — CHANGES_REQUESTED** [review]
Should be in a subdir containing version ie eido/0.1.6/Dockerfile

**Comment by osallou** [comment]
Biotools id is eido-python-package according to biotools web site

**Comment by osallou** [comment]
Did you test the container? I see you just unzip the release , but i see no python install nor requirements installation, wonder if it works....

Why not just installing the python package via pip (as root)?

Something like

USER root
RUN pip install eido==0.1.6 (pip or pip3 according to image ..)
USER biodocker

**Comment by rafalstepien** [comment]
@osallou I updated and tested the container

File: eido/0.1.6/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# Metadata
+LABEL base_image="biocontainers:latest"
+LABEL version="1"
+LABEL software="eido"
+LABEL software.version="0.1.6"
+LABEL about.summary="An open source tool used to validate and convert PEP format sample metadata against a schema."
+LABEL about.home="http://eido.databio.org/en/latest/"
+LABEL about.license="BSD-2-Clause"
+LABEL about.tags="PEP, metadata, validation"
+LABEL extra.identifiers.biotools=eido
```
Comment (osallou) [inline eido/0.1.6/Dockerfile]: In biotools, id is eido-python-package

**Review by osallou — CHANGES_REQUESTED** [review]

**Comment by osallou** [comment]
@rafalstepien merged, thanks for contrib and patience

**Pull request merged**